src/store/authors/reducer.js

```diff
@@ -0,0 +1,39 @@
+import {
+	ADD_COURSE_AUTHOR,
+	CREATE_AUTHOR,
+	REMOVE_COURSE_AUTHOR,
+	RESET_COURSE_AUTHORS,
+	SAVE_AUTHORS,
+} from './actionTypes';
+
+const authorsInitialState = { authors: [], courseAuthors: [] };
+
+export default function authorsReducer(
+	state = authorsInitialState,
+	{ type, payload }
+) {
+	switch (type) {
+		case SAVE_AUTHORS: {
```

Comment:
SAVE_AUTHORS is basically GET_AUTHORS
![image](https://user-images.githubusercontent.com/28926420/154776682-09442724-f15e-4160-a05e-f38ea2c49f95.png)

Reply:
Got it